source/dotnet/Library/AdaptiveCards.Templating.CSharp.WinRT/AdaptiveCards.Template.csproj

```diff
@@ -21,10 +21,6 @@
 
   <PropertyGroup>
     <CsWinRTComponent>true</CsWinRTComponent>
-    <SignAssembly>True</SignAssembly>
```

Comment:
How was this solved? Does it require a matching change in the Azure repo?

Reply:
I didn't necessarily fixed the issue, I took the reference from Paul's OB change.